README.md

```diff
@@ -54,5 +54,31 @@ will begin to drift slightly from the original design. This is expected and
 there is no expectation that the original design proposal needs to be updated
 to reflect these differences.
 
-The code and our documentation are the ultimate sources of truth. CFPs are merely
-the starting point for the implementation.
+The code and our documentation are the ultimate sources of truth. CFPs are
+the starting point for the implementation, but can also be updated as needed.
+
+# Status
+
+The status of a CFP indicates its maturity. There are four different statuses
+that a CFP can have:
+
+1. Draft: A CFP in any form (Google doc, Github issue, ect.) that has not yet
+been merged into this repository.
+
+2. Implementable: CFPs that have been approved by one committer and merged into
+this repository are given the implementable status. This means that the ideas in the CFP
+have been agreed in principle and the coding work can now be started. When merged,
+the CFP should be up to date and the relevant stakeholders should be in allignement
+even if they are still going through the practical ramifications of how to implement it.
+
+4. Updated: If there have been signifigant changes been the original CFP and the
+implementation, the CFP can be updated to reflect those changes. The new status of
+the CFP is Updated since it has changed signifigantly since it was first merged.
+
+5. Dormant: This status is given to CFP that have not or did not yet end up being
+implemented for a variety of reasons, like not enough engineering cycles or not
+currently a focus on of the project. This can be either CFP that were discussed
+but not implemented and the project would like to preserve those discussions for
+future use or CFPs that were deemed implementable, but no one ever got around
+to implementing them and the CFP is now dormant. Dormant CFPs can be reactivated
+at any time if there is interest in the project.
```

Comment:
Could perhaps add something to the effect that "The next step for a dormant CFP is to amend it to be Implementable."

Reply:
added